### PR TITLE
#185172115 a buyer should track order status

### PR DESCRIPTION
**Pull request by Pacific911**
#### What does this PR do?

  - A buyer should be to track the status of the orders 
  - A buyer should be to check the status of the product if it is confirmed or still pending

#### How should this be manually tested?
 - GET : /orders

#### What are the relevant pivotal tracker/Trello stories?
 - 185172115

#### Screenshots (if appropriate)
    - A User can review all the order history page
    
whole
   
    - A User when he/she clicks on the product should be taken to an order detail page displaying the order status
  
single product

### CODE REVIEW

**Review by JosephNgabo — COMMENTED** [review]
try to resolve some conflicts available due to rebase

**Review by princoo — CHANGES_REQUESTED** [review]
on your page you need to provide an `element(button)` that redirects to the payment form
if the order is `not yet` payed successfully (does not have status `succeeded`).

File: src/pages/orders/redux/trackOrderSlice.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+//     orders: {
+//       id: string;
+//       status: string;
+//       products: {
+//         product: Products;
+//         quantity: number;
+//       }[];
+//     }[];
+//   };
+// }
```
Comment (munezerobagira) [inline src/pages/orders/redux/trackOrderSlice.tsx:19]:
```suggestion
```

File: src/pages/orders/trackOrder.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { useEffect } from 'react';
+import { useGetOrderStatus } from './redux/hooks';
+
+function Orders() {
+  const { handleGetOrderStatus, result } = useGetOrderStatus();
+  useEffect(() => {
+    handleGetOrderStatus();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (munezerobagira) [inline src/pages/orders/trackOrder.tsx:8]:
```suggestion
```

**Review by munezerobagira — CHANGES_REQUESTED** [review]

File: src/pages/orders/redux/trackOrderSlice.tsx (hunk 1 above)
Comment (Pacific911) [reply src/pages/orders/redux/trackOrderSlice.tsx]: done 👍

File: src/pages/orders/trackOrder.tsx (hunk 2 above)
Comment (Pacific911) [reply src/pages/orders/trackOrder.tsx]: solved

**Review by princoo — COMMENTED** [review]
adjust the position of elements in this column

order

**Review by Edcreation — APPROVED** [review]

**Review by AdrineUWERA — APPROVED** [review]

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**